src/app/public/modules/list-view-grid/list-view-grid.component.ts

```diff
@@ -141,6 +150,23 @@ export class SkyListViewGridComponent
   }
 
   public ngAfterContentInit() {
+
+    // Watch for selection changes and update _multiselectSelectedIds for local comparison.
+    this.state.map(s => s.selected.item)
+      .takeUntil(this.ngUnsubscribe)
+      .distinctUntilChanged(this.selectedMapEqual)
+      .subscribe((items: ListSelectedModel) => {
+        let selectedIds: string[] = [];
```

Comment:
`selectedIds` value never changes; can be `const`.

Reply:
You sure? I'm pushing new values to it in the following `.foreach()`

Reply:
The contents of the array change, but the array itself is never reassigned.

Reply:
Well I learned something today!
https://stackoverflow.com/questions/23436437/why-can-i-change-value-of-a-constant-in-javascript

Fixed.